src/Platforms/PostgreSQLPlatform.php

```diff
@@ -809,6 +810,29 @@ public function getDropForeignKeySQL($foreignKey, $table)
         return $this->getDropConstraintSQL($foreignKey, $table);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getDropIndexSQL($index, $table = null)
+    {
+        /** @psalm-suppress MissingClosureParamType */
+        $tableName = static fn ($t): string => $t instanceof Table ? $t->getName() : (string) $t;
```

Comment:
Let's make this a private method. This allows us to document the parameter type via PHPDoc which should make Psalm happy.